planetiler-core/src/main/java/com/onthegomap/planetiler/Planetiler.java

```diff
@@ -515,6 +517,13 @@ public void run() throws Exception {
       return; // exit only if just fetching wikidata or downloading sources
     }
 
+    String osmPolyFilePath =
+      arguments.getString("poly_shape", "a .poly file defining the \"bounds\" to generate", null);
```

Comment:
Let's add one integration test in `PlanetilerTests` that uses this option and ensures it's doing what we expect.  We want to make sure that 1) the options works without throwing exceptions and 2) points, lines, polygons, and polygon fills don't show up outside the bounding shape. If the integration test gets too complex verifying the cases for 2 then you could add some unit tests to `FeatureRendererTest` to verify the different point, line, polygon, and polygon fill variations.